src/test/java/uk/gov/hmcts/darts/common/util/CommonTestDataUtil.java

```diff
@@ -561,6 +561,14 @@ public TranscriptionUrgencyEntity createTranscriptionUrgencyEntityFromEnum(Trans
         return transcriptionUrgencyEntity;
     }
 
+    public TranscriptionStatusEntity createTranscriptionStatusEntityFromEnum(TranscriptionStatusEnum statusEnum) {
+        TranscriptionStatusEntity transcriptionStatusEntity = new TranscriptionStatusEntity();
+        transcriptionStatusEntity.setId(statusEnum.getId());
+        transcriptionStatusEntity.setDisplayName(statusEnum.name());
+
+        return transcriptionStatusEntity;
+    }
+
```

Comment:
Should this grab the value from the database assuming TranscriptionStatusEntity are constants?